cdap-kafka/src/main/java/co/cask/cdap/kafka/run/KafkaServerMain.java

```diff
@@ -150,10 +150,6 @@ private Properties generateKafkaConfig(int brokerId, String zkConnectStr, String
     Map<String, String> propConfigs = cConf.getValByRegex("^(kafka\\..*)$");
     for (Map.Entry<String, String> pair : propConfigs.entrySet()) {
       String key = pair.getKey();
-      if(key.equals(KafkaConstants.ConfigKeys.PORT_CONFIG_DEPRECATED) || key.equals(KafkaConstants.ConfigKeys.ZOOKEEPER_NAMESPACE_CONFIG_DEPRECATED)
```

Comment:
because it has the potential to be confusing logically, and it simply would mean these three properties are set again.